Examples/src/main/kotlin/com/mapbox/vision/examples/SafetyActivityKt.kt

```diff
@@ -72,6 +77,8 @@ class SafetyActivityKt : BaseActivity() {
 
         override fun onRoadRestrictionsUpdated(roadRestrictions: RoadRestrictions) {
             maxAllowedSpeed = roadRestrictions.speedLimits.car.max
+            // set speed value to view to show it if overspeeding occurs
```

Comment:
`set speed value to view to show if overspeeding occurs` ?

Reply:
OK